hs/src/Reach/EPP.hs

```diff
@@ -171,6 +171,8 @@ data BEnv = BEnv
   , be_view_setsr :: IORef ViewSet
   , be_inConsensus :: Bool
   , be_counter :: Counter
+  , be_which :: Int
+  , be_api_info :: IORef (M.Map SLPart ([DLType], (Maybe String), Int))
```

Comment:
Here's a place where that named typed will come in handy. I think it would live in DLBase